scripts/exregional_run_post.sh

```diff
@@ -114,6 +114,10 @@ case $MACHINE in
     ;;
 
   "ORION")
+    ulimit -s unlimited
+    ulimit -a
+    export OMP_NUM_THREADS=1
+    export OMP_STACKSIZE=1024M
     APRUN="srun"
```

Comment:
This is very different than the requirements for the other machines. It would be good to leave a comment as to why this is the case.

Reply:
I put this here simply because it makes job run through. We have limited experience with Orion. We can tune this if needed.